### ISSUE TITLE
Feature Request: generate a "tvshow.nfo" file in the directory when "write NFO" is enabled

### ISSUE THREAD

**Issue by PavelGutin** [meeb/tubesync#302]
In order for Kodi to be able to add a show to the library, it needs to have a `tvshow.nfo` in the directory. It would be nice if this was done automatically. More details are in the Kodi wiki https://kodi.wiki/view/NFO_files/TV_shows

**Comment by jctennis** [comment]
I second the request for this feature. I have been creating them manually but having tubesync create one just using the channel description and logo would be helpful. 

Here is an example of one I made manually for https://www.youtube.com/playlist?list=PLib8CA6AKJM8KLkWy4JGGYP0d4E-JYZOa

`
<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>

    <tvshow>
            <title>SNES Drunk</title>
            <showtitle>SNES Drunk</showtitle>
            <plot>One person making short, to-the-point reviews of retro games. Super Nintendo, Sega Genesis, Game Boy, Arcade, PC, NES, GBA, and more.</plot>
            <genre>Gaming</genre>
            <premiered>2013-05-31T23:31:48Z</premiered>
            <aired>2013-05-31T23:31:48Z</aired>
            <studio>SNES Drunk</studio>
            <thumb>https://yt3.ggpht.com/ytc/AMLnZu8JVhMO5QhUVKNZXfX6lwr0aRYnF5SMA0e8_UeDJg=s800-c-k-c0x00ffffff-no-rj</thumb>
            <thumb aspect="poster">https://yt3.ggpht.com/ytc/AMLnZu8JVhMO5QhUVKNZXfX6lwr0aRYnF5SMA0e8_UeDJg=s800-c-k-c0x00ffffff-no-rj</thumb>
            <thumb aspect="banner">https://yt3.ggpht.com/ytc/AMLnZu8JVhMO5QhUVKNZXfX6lwr0aRYnF5SMA0e8_UeDJg=s800-c-k-c0x00ffffff-no-rj</thumb>
            <fanart>
                    <thumb>https://yt3.ggpht.com/ytc/AMLnZu8JVhMO5QhUVKNZXfX6lwr0aRYnF5SMA0e8_UeDJg=s800-c-k-c0x00ffffff-no-rj</thumb>
            </fanart>
            <tag>Youtube</tag>
    </tvshow>`

**Comment by bonelifer** [comment]
This would definitely be nice.

**Comment by tcely** [comment]
The channel information isn't immediately available for playlists, it requires another call of `yt_dlp` to fetch those details.

I'm considering a post processor script that can be called after the playlist is downloaded. It would only create a minimal `tvshow.nfo` file in the source directory.

```
<tvshow>
    <title>[The `name` field from the Source]</title>
    <uniqueid type="Youtube" default="true">[The `key` field from the Source]</uniqueid>
</tvshow>
```

Any thoughts? @bonelifer @jctennis @PavelGutin

**Comment by jctennis** [comment]
I like it

**Comment by tcely** [comment]
Ok.

<details>
<summary>Download the command file from within an older container</summary>

We can download[^1] the new file from #1103, with `curl`:
[^1]: `docker pull ghcr.io/meeb/tubesync:latest` will make using `curl` from inside the container unnecessary.

```sh
$ curl -L -o /app/sync/management/commands/create-tvshow-nfo.py \
    'https://github.com/meeb/tubesync/raw/refs/heads/main/tubesync/sync/management/commands/create-tvshow-nfo.py'
```

</details>

Then add the post processor in `local_settings.py` to give this a try:

```python
# This is simply copied from `settings.py` with 
# an added `postprocessors` key at the top.
YOUTUBE_DEFAULTS = {
    'postprocessors': list((
        dict(
            key='Exec',
            when='playlist',
            exec_cmd="/usr/bin/env python3 /app/manage.py create-tvshow-nfo '%(id)s'",
        ),
        # another postprocessor would be a new dict
    )),
    'color': 'never',       # Do not use colours in output
    'age_limit': 99,        # 'Age in years' to spoof
    'ignoreerrors': True,   # Skip on errors (such as unavailable videos in playlists)
    'cachedir': False,      # Disable on-disk caching
    'addmetadata': True,    # Embed metadata during postprocessing where available
    'geo_verification_proxy': getenv('geo_verification_proxy').strip() or None,
    'max_sleep_interval': (60)*5,
    'sleep_interval': 0.25,
}
```

**Comment by tcely** [comment]
After #1103 was merged, using the `latest` image, you can skip downloading the new command with `curl` inside the container.

```
docker pull ghcr.io/meeb/tubesync:latest
```

@jctennis Please open a new issue if you need any further changes. Thanks.
